src/pages/Index.vue

```diff
@@ -180,6 +189,7 @@ export default defineComponent({
     const ddosEnabled = ref(true)
     const forceProxy = ref(true)
     const atackCounter = ref(0)
+    const atackErrorCounter = ref(0)
```

Comment:
But change to `attackErrorCounter`, atack is typo.

Reply:
@Andrii-L-DDoSer I know its typo, but everywhere in the file `atack` is used(